source/strings.tex

```diff
@@ -3009,8 +3009,8 @@
 \effects
 A directive that informs a \tcode{basic_string} of a planned change in size,
```

Comment:
(Not sure what's going on here with directives and informing objects, but that's presumably some vintage wording that we'll best leave as is.)

Reply:
:see_no_evil: